src/components/ArticleList.jsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { useState, useEffect } from "react";
+import { getArticleList } from "../utils/getArticleList";
+
+const ArticleList = () => {
+  const [articleList, setArticleList] = useState([{}]);
+
+  useEffect(() => {
+    getArticleList().then((apiResult) => {
+      setArticleList(apiResult);
+    });
+  }, []);
+
+  return (
+    <section className="ArticleList">
+      {/* <span>02</span> */}
+      <ul className="Article-list">
+        {articleList.map((article) => {
+          return (
+            <li className="Article--card" key={article.article_id}>
+              <h1>{article.topic}</h1>
+              <h2>{article.title}</h2>
+              <p>
+                {article.author} | {article.created_at.slice(0, 10)}
```

Comment:
The slice doesn't seem to work, we are getting an error from this and it won't allow page to load. Please double check this syntax  and ensure it works. When we removed this line, the page loaded